source/manual/zendesk.html.md

```diff
@@ -23,7 +23,10 @@ relevant and actionable - and start with the oldest by date. If external assista
 line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
 2nd line queue).
 
-Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+'Low' priority tickets are for tickets that are:
+* `on-hold` or are waiting to be handed over to another team (unless they are product related, they
+  can be assigned to `3rd Line--GOV.UK Product Requests`)
```

Comment:
>can be assigned to `3rd Line--GOV.UK Product Requests`)

I don't think this is needed? Tickets should simply be assigned, not left in Low